coalib/coala.py

```diff
@@ -35,9 +33,12 @@ def main():
         args = default_arg_parser().parse_args()
 
         if args.show_bears:
+            from coalib.settings.ConfigurationGathering import (
+                get_filtered_bears)
             local_bears, global_bears = get_filtered_bears(
                 args.filter_by_language, log_printer)
 
+            from coalib.output.ConsoleInteraction import show_bears
             show_bears(local_bears,
                        global_bears,
                        args.show_description or args.show_details,
```

Comment:
So you do `coala --show-capabilities Python` and it will list stuff like `Code Simplification, security, ...` and stuff. Basically a union of all `CAN_FIX`es of all python bears.